lib/hg-utils.coffee

```diff
@@ -242,7 +242,11 @@ class Repository
       params = []
     if !util.isArray(params)
       params = [params]
-    child = spawnSync('hg', params)
+
+    return '' unless @isCommandForRepo(params)
+
+    options = { cwd: @rootPath }
+    child = spawnSync('hg', params, options)
```

Comment:
I already had mercurial installed, but I restarted anyway. Error is still there. Although the full path is there, it probably needs the cwd being specified so the command knows where the .hg file is???